app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+
+
+from app.errors import (NotVaccinatedError,
+                        NotWearingMaskError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("All friends should be vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("All friends should be vaccinated")

```

Comment:
it seems to me, that massages should be different for different exceptions)

Reply:
I also wanted to do this, but the tests expect the same message for both of these cases. It should not be?

Reply:
![image](https://user-images.githubusercontent.com/123042088/220133436-1383d8ea-b6f5-46ad-8bcc-f6c53eab9e1b.png)